### ISSUE TITLE
Image-Text Retrieval Task, ITC score for ranking

### ISSUE THREAD

**Issue by yxoh** [salesforce/ALBEF#74]
I saw the original setting use the ITM score s_{itm} for ranking, but it has more calculations. Is it ok that we only use feature similarity score s_{itc} for ranking during inference?

**Comment by LiJunnan1992** [comment]
Yes, using ITC is faster but less accurate.

**Comment by yxoh** [comment]
> ut less accurate

Are there experiments that show how much the accuracy rate has dropped?

**Comment by yxoh** [comment]
> > ut less accurate
> 
> Are there experiments that show how much the accuracy rate has dropped?

Ah, I saw the experiment in the paper:)

**Comment by yxoh** [comment]
I downloaded the data JSON files and pretrained ALBEF model (4M) from this repo. I run an image-text retrieval task. The zero-shot results on the flickr30k dataset are TR (R@1: 84.9, R@5: 97.2, R@10:99.0); IR (R@1: 68.18, R@5: 88.58, R@10: 93.02). But in the paper, the results are TR (R@1: 90.5, R@5: 98.8, R@10:99.7); IR (R@1: 76.8, R@5: 93.7, R@10: 96.7).
How can I reproduce the same results in the paper?

**Comment by LiJunnan1992** [comment]
The flickr zero-shot results are obtained using the coco-finetuned model

**Comment by yxoh** [comment]
> The flickr zero-shot results are obtained using the coco-finetuned model

It helps me. Thanks:)

**Comment by zzzzzigzag** [comment]
> > > ut less accurate
> > 
> > 
> > Are there experiments that show how much the accuracy rate has dropped?
> 
> Ah, I saw the experiment in the paper:)

hi，@[yxoh](https://github.com/yxoh) could you show where exactly the experiment results you mentioned are in the paper? I did not find it, thanks a lot!
